docs/fsharp/language-reference/query-expressions.md

```diff
@@ -537,11 +537,14 @@ query {
 <td><code>EXISTS</code>
 <br />
 
+<!-- markdownlint-capture -->
```

Comment:
Ideally, these would be proper Markdown code blocks and the content would be ignored automatically. HTML content isn't treated specially, so disable/restore the relevant rule here.

Reply:
cc @cartermp I think that would be a better fix. It looks like all the languages being displayed are supported.

Reply:
I didn’t want to completely rewrite this file as part of a targeted PR. Are you asking me to make that change or noting that it would be a good item for the backlog?

Reply:
> Are you asking me to make that change or noting that it would be a good item for the backlog?

Let's add it to the backlog. I agree with keeping this more focused. /cc @cartermp